universal-application-tool-0.0.1/app/views/admin/questions/QuestionConfig.java

```diff
@@ -111,12 +111,14 @@ private QuestionConfig addAddressQuestionConfig(AddressQuestionForm addressQuest
             .setId("address-question-disallow-po-box-checkbox")
             .setFieldName("disallowPoBox")
             .setLabelText("Disallow post office boxes")
+            .setValue("true")
```

Comment:
I think this should be `setValue(addressQuestionForm.getDisallowPoBox())`.

Reply:
I don't actually know the details, but it looks like this is working as intended (see the screenshots in the description). The form data in the request object does not include `disallowPoBox` at all when the box isn't checked, and includes it with `"true"` when it is checked.

Reply:
Thank you for explaining! That makes sense.